assets/css/styles.css

```
@@ -50,3 +50,4 @@ body {
 @import "./components/hextra/feature-grid.css";
 @import "./components/jupyter.css";
 @import "./components/badge.css";
+@import "./components/toc.css";
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)